src/azure_cosmosdb/cosmosdb_data_mappings.bal

```diff
@@ -0,0 +1,36 @@
+import ballerina/http;
+
+isolated function mapParametersToHeaderType(string httpVerb, string url) returns RequestHeaderParameters {
+    RequestHeaderParameters params = {};
+    params.verb = httpVerb;
+    params.resourceType = getResourceType(url);
+    params.resourceId = getResourceId(url);
+    return params;
+}
+
+isolated function mapResponseHeadersToObject(http:Response|http:ClientError httpResponse) returns @tainted Headers|error 
+{
```

Comment:
Shall we move this open curly bracket to the previous line? 
Check in all the places and fix similar issue